source/IAccessibleHandler/internalWinEventHandler.py

```diff
@@ -144,6 +164,19 @@ def winEventCallback(handle, eventID, window, objectID, childID, threadID, times
 			# WM_NULL to this window at this point (which happens in accessibleObjectFromEvent), Messenger will
 			# silently exit (#677). Therefore, completely ignore these events, which is useless to us anyway.
 			return
+		if config.conf["iaccessible"]["specificObjEvents"] and not (

```

Comment:
For UIA, the config section name is capitalized. I think that makes sense. We also do it for the IAccessible NVDAObjects package.
```suggestion
		if config.conf["IAccessible"]["specificObjEvents"] and not (

```